buildscripts/common.props

```diff
@@ -2,6 +2,7 @@
 	
 	<PropertyGroup>
 		<NoWarn>$(NoWarn);CS1591;CS3014;CS3003;CS3001;CS3021</NoWarn>
+		<NoWarn>$(NoWarn);CS0612;CS0618</NoWarn> <!-- TODO: Remove this line once `[Obsolete]` members have been dealt with. -->
```

Comment:
I am not a fan of disabling `[Obsolete]`-related warnings CS0612 and CS0618 solution-wide, but this still seems better than sprinkling `#pragma warning disable`s and `#pragma warning restore`s across the whole code base. `[assembly: SuppressMessage]` would have been nice, but I couldn't get it to work with these warnings.

We should remove this again right before releasing Castle Core 5.0.0.